### PR TITLE
Update release procedure

### PR DESCRIPTION
**Pull request by masa213f**
Signed-off-by: Masayuki Ishii <masa213f@gmail.com>

### CODE REVIEW

**Comment by masa213f** [comment]
The e2e test has remained broken by #9.

File: docs/release.md [hunk 1]
```diff
@@ -30,7 +30,14 @@ It should look like:
 
 ## Bump version
 
-1. Determine a new version number.  Let it write `$VERSION` as `VERSION=x.y.z`.
+1. Determine a new version number. Then set `VERSION` variable.
+
+    ```console
+    # Set VERSION and confirm it. It should not have "v" prefix.
+    $ VERSION=x.y.x
```
Comment (kmdkuk) [inline docs/release.md:37]:
```suggestion
    $ VERSION=x.y.z
```

File: docs/release.md [hunk 2]
```diff
@@ -49,9 +56,17 @@ It should look like:
 6. Add a git tag to the main HEAD, then push it.
 
     ```console
+    # Set VERSION again.
+    $ VERSION=x.y.x
```
Comment (kmdkuk) [inline docs/release.md:60]:
```suggestion
    $ VERSION=x.y.z
```

**Review by kmdkuk — CHANGES_REQUESTED** [review]

**Review by kmdkuk — APPROVED** [review]

**Comment by kmdkuk** [comment]
Test failure is a known issue.
Merge as it is not related to this PR.

https://github.com/cybozu-go/nyamber/pull/11#issuecomment-1182705840

**Pull request merged**